tests/std/tests/GH_003105_piecewise_densities/test.cpp

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <random>
+
+int main() {
+    (void) std::piecewise_constant_distribution<float>{}.densities();

```

Comment:
We could easily test `intervals()` (a similar member function also returning `vector<result_type>`) and `piecewise_linear_distribution` to be thorough...

... which revealed that `piecewise_linear_distribution` is affected by the same issue!